tutorials/test_volume_source_space.py

```diff
@@ -0,0 +1,67 @@
+import numpy as np  # noqa
+from mayavi import mlab  # noqa
+
```

Comment:
Errant commit to include this?

Reply:
Working on finding where does this thing come from

Reply:
Fixed